packages/xo-web/src/common/store/actions.js

```diff
@@ -58,3 +58,7 @@ export const setHomeVmIdsSelection = createAction(
   'SET_HOME_VM_IDS_SELECTION',
   homeVmIdsSelection => homeVmIdsSelection
 )
+export const setHubInstallLoadingState = createAction(
+  'SET_HUB_INSTALL_LOADING_STATE',
+  hubInstallLoadingState => hubInstallLoadingState
+)
```

Comment:
I think this method could higher level (or you can even declare multiple methods):

```js
markHubResourceAsInstalling(id)
markHubResourceAsInstalled(id)
```

> The names are far from perfect, feel free to choose a better one.